### PR TITLE
Add test case for ClientSetFromFile in cmd/kubeadm/app/util/kubeconfig package

### PR DESCRIPTION
**Pull request by liubin**
**What this PR does / why we need it**:

Add test case to cmd/kubeadm/app/util/kubeconfig

Improved coverage from 40% to 70% with `make test KUBE_COVER=y WHAT=./cmd/kubeadm/app/util/kubeconfig`

**Release note**:

```release-note
None
```

### CODE REVIEW

**Comment by dixudx** [comment]
/ok-to-test

@liubin Please update the PR title to be more specific. Thanks.

File: cmd/kubeadm/app/util/kubeconfig/kubeconfig_test.go [hunk 1]
```diff
@@ -178,3 +178,45 @@ func TestWriteKubeconfigToDisk(t *testing.T) {
...
+			rt.cc.caCert,
+			rt.ccWithToken.token,
+		)
+		configPath := fmt.Sprintf("%s/etc/kubernetes/%s.conf", tmpdir, rt.name)
+		if err := WriteToDisk(configPath, c); err != nil {
+			t.Errorf("Write kubeconf to file got an unexception error: %s", err.Error())
+		}
+
+		_, err = ClientSetFromFile(configPath)
+		if (err != nil && rt.expected == nil) ||
```
Comment (dixudx) [inline cmd/kubeadm/app/util/kubeconfig/kubeconfig_test.go]: IMO this condition can be re-ordered.

File: cmd/kubeadm/app/util/kubeconfig/kubeconfig_test.go [hunk 2]
```diff
@@ -178,3 +178,45 @@ func TestWriteKubeconfigToDisk(t *testing.T) {
...
+	if err != nil {
+		t.Fatalf("Couldn't create tmpdir")
+	}
+	defer os.RemoveAll(tmpdir)
+
+	var writeConfig = []struct {
+		name        string
+		cc          configClient
+		ccWithToken configClientWithToken
+		expected    error
```
Comment (kad) [inline cmd/kubeadm/app/util/kubeconfig/kubeconfig_test.go]: I'd rename `expected` to `expectedError` or something similar, to get `if (err != nil && rt.expected == nil) || ... ` lines more readable.

**Pull request closed without merging**